File: p006.py
# 
# Solution to Project Euler problem 6
# by Project Nayuki
# 
# http://www.nayuki.io/page/project-euler-solutions
# https://github.com/nayuki/Project-Euler-solutions
# 


def compute():
	sum = 0
	sum2 = 0
	for i in range(1, 101):
		sum += i
		sum2 += i**2
	return str(sum**2 - sum2)


if __name__ == "__main__":
	print(compute())

File: java/p006.java
/* 
 * Solution to Project Euler problem 6
 * Copyright (c) Project Nayuki. All rights reserved.
 * 
 * https://www.nayuki.io/page/project-euler-solutions
 * https://github.com/nayuki/Project-Euler-solutions
 */


public final class p006 implements EulerSolution {
	
	public static void main(String[] args)
    {
        int sumOfSquares = 0;
        int sum = 0;
        for (int i = 1; i <= 100; i++)
        {
            int square = i*i;
            sumOfSquares += square; 
        }
        for (int i = 1; i <= 100; i++)
        {
            sum += i;
        }

        int total = sum*sum;
        System.out.println(total -= sumOfSquares);
    }
	
}
